lib/lifecycles/changelog.js

```diff
@@ -60,3 +63,15 @@ function createIfMissing (args) {
     }
   }
 }
+
+function loadContext (contextPath) {
+  var context
+  try {
+    accessSync(contextPath, fs.F_OK)
+    context = require(contextPath)
```

Comment:
Would it be better to use `JSON.parse(fs.readFileSync(contextPath, 'utf8'))` instead of a "dynamic" require?